src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,70 @@
+package duke.task;
+
+import duke.DukeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.FormatStyle;
+
+public class Task {
+
+    private String description;
+    private boolean isDone;
+    private LocalDate date = null;
+
+    public Task(String description, boolean done) {
+        this.description = description;
+        this.isDone = done;
+    }
+
+    public Task(String description, String date, boolean done) throws DukeException {
```

Comment:
If I am not wrong, we still need to include Javadocs for public constructors. We can only ignore Javadocs for public getters and setters methods.
See: https://se-education.org/guides/conventions/java/intermediate.html#comments